.github/dependabot.yml

```diff
@@ -22,3 +22,10 @@ updates:
     - 6.1.2
     - 6.1.2.1
     - 6.1.3
+
+- package-ecosystem: "docker"
+  directory: "/"
+  schedule:
+      interval: "daily"
+      time: "03:00"
```

Comment:
The convention with YAML is to use two space characters for indented levels, and to use consistent indenting within a YAML file. Can you change these two lines to indent by 2 characters rather than 4.

Reply:
Done. I've adjusted the indentation in the Dependabot Docker configuration to follow the YAML convention